evalml/data_checks/highly_null_data_check.py

```diff
@@ -47,7 +47,7 @@ def validate(self, X, y=None):
                                                                  "code": "HIGHLY_NULL",\
                                                                  "details": {"column": "lots_of_null"}}],\
                                                     "actions": [{"code": "DROP_COL",\
-                                                                 "metadata": {"column": "lots_of_null"}}]}
+                                                                 "metadata": {"columns": "lots_of_null"}}]}
```

Comment:
@angela97lin this is a breaking change, yes? Could you please mention it in the release notes?